### PR TITLE
Add sample plugin docker config

### PR DESCRIPTION
**Pull request by karelhala**
### Description

We need to have an option to spin the sample plugin with docker, this PR adds docker config with option to pass custom variables to it in order to serve the plugin on HCC.

### CODE REVIEW

File: packages/sample-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Sample plugin
+
+### Docker config
+
+There's docker config with caddy server for easier build and run of the application. The Caddy config utilizes two environment variables
```
Comment (florkbr) [inline packages/sample-plugin/README.md:5]: @karelhala should we refer to this as the `plugin` and not an `application`? @vojtechszocs any input on the readme? I know you were going to look at adding them for the sample app and plugin.

File: packages/sample-plugin/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM quay.io/cloudservices/caddy-ubi:357c825
+
+COPY ./Caddyfile /opt/app-root/src/Caddyfile
+COPY dist /opt/app-root/src/dist/
+COPY ./package.json /opt/app-root/src
+WORKDIR /opt/app-root/src
+ENV PLUGIN_URL=/
+ENV FALLBACK_URL=/
```
Comment (BlakeHolifield) [inline packages/sample-plugin/Dockerfile:8]: I like the ENV idea here :+1:

**Review by BlakeHolifield — APPROVED** [review]
Looks good from my end. Nice Caddyfile upgrade!

File: packages/sample-plugin/README.md (hunk 1 above)
Comment (karelhala) [reply packages/sample-plugin/README.md:5]: Good point! Will change it to `plugin`

**Pull request merged**